client/src/components/Button/Button.tsx

```diff
@@ -6,11 +6,14 @@ import './Button.scss'
 export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   className?: string
   children?: string
+  ref?: React.RefObject<HTMLButtonElement>
 }
 
-const Button: FC<ButtonProps> = ({ className, ...other }) => {
+const Button: FC<ButtonProps> = ({ className, ...other }, ref) => {
```

Comment:
We might as well get rid of the intermediary variable `Button` and directly wrap our function into `forwardRef`.
```suggestion
const Button = forwardRef<HTMLButtonElement, ButtonProps>(({ className, ...other }, ref) => {
```